crates/turbopack-node/js/src/transforms/webpack-loaders.ts

```diff
@@ -41,6 +41,66 @@ const toPath = (file: string) => {
   return sep !== "/" ? relPath.replaceAll(sep, "/") : relPath;
 };
 
+const LogType = Object.freeze({
+  error: "error",
+  warn: "warn",
+  info: "info",
+  log: "log",
+  debug: "debug",
+
+  trace: "trace",
+
+  group: "group",
+  groupCollapsed: "groupCollapsed",
+  groupEnd: "groupEnd",
+
+  profile: "profile",
+  profileEnd: "profileEnd",
+
+  time: "time",
+
+  clear: "clear",
+  status: "status",
+});
+
+const loaderFlag = "LOADER_EXECUTION";
+
+const cutOffByFlag = (stack, flag) => {
+  const errorStack = stack.split("\n");
+  for (let i = 0; i < errorStack.length; i++) {
+    if (errorStack[i].includes(flag)) {
+      errorStack.length = i;
+    }
+  }
+  return errorStack.join("\n");
+};
+
+/**
+ * @param {string} stack stack trace
+ * @returns {string} stack trace without the loader execution flag included
+ */
+const cutOffLoaderExecution = (stack) => cutOffByFlag(stack, loaderFlag);
```

Comment:
Can you explain why this is needed?

Reply:
These are copy-paste from what webpack does for the logging with its levels. We could not do it, but tried to mimic its behavior as much.

Reply:
ref: https://github.com/webpack/webpack/blob/81c8fbde2e0e4b8d2e30a5109dedc082ef8c09dc/lib/Compilation.js#L1166